packages/admin/src/pages/build-application/[applicationId]/error-multiple-editors.page.tsx

```diff
@@ -0,0 +1,70 @@
+import { GetServerSidePropsContext } from 'next';
+import Link from 'next/link';
+import InferProps from '../../../types/InferProps';
+import {
+  getApplicationFormSummary,
+  getLastEditedEmail,
+} from '../../../services/ApplicationService';
+import Meta from '../../../components/layout/Meta';
+import { getSessionIdFromCookies } from '../../../utils/session';
+import { formatDateTimeForSentence } from '../../../utils/dateFormatterGDS';
+
+export const getServerSideProps = async (
+  context: GetServerSidePropsContext
+) => {
+  const sessionCookie = getSessionIdFromCookies(context.req);
+  const { applicationId } = context.params as Record<string, string>;
+
+  const application = await getApplicationFormSummary(
+    applicationId,
+    sessionCookie,
+    false,
+    false
+  );
+  const {
+    audit: { lastUpdated },
+  } = application;
+
+  const editorOrPublisherEmail = await getLastEditedEmail(
+    applicationId,
+    sessionCookie
+  );
```

Comment:
We may want to add error handling for these two service calls. It will just whitescreen atm